tests/ui/parser/deli-ident-issue-2.stderr

```diff
@@ -4,16 +4,7 @@ error: mismatched closing delimiter: `]`
 LL |     if 1 < 2 {
    |              ^ unclosed delimiter
 LL |         let _a = vec!];
-   |                      ^ mismatched closing delimiter
+   |                      ^ mismatched closing delimiter, may missing open `[`
```

Comment:
for example, this is the scenario we should report missing open `[`, because except for this `]` is lefted unmistached, all the other outer delimiter are matched correctly.